README.md

```diff
@@ -3,6 +3,28 @@ This operator is the primary operator for Open Data Hub. It is responsible for e
 Jupyter Notebooks, Modelmesh serving, Datascience pipelines etc. The operator makes use of `DataScienceCluster` CRD to deploy
 and configure these applications.
 
+### Table of contents
+- [Usage](#usage)
+  - [Installation](#installation)
+- [Dev Preview](#dev-preview)
```

Comment:
I think `Dev Preview` should be sibling not parent tag. There is a separate link for Dev readme file.

Reply:
Thanks for the review @ajaypratap003  👍 . so When I see the Dev Preview markdown it has `## Dev Preview`
Which is on the same level as Usage right? Please correct me if I am wrong

Reply:
If it's intentionally mark as parent then fine.